tests/lib/rules/jsx-sort-default-props.js

```diff
@@ -314,12 +314,17 @@ ruleTester.run('jsx-sort-default-props', rule, {
     ].join('\n')
   }, {
     code: [
+      'const defaults = {',
+      '  b: "b"',
+      '};',
       'const First = (props) => <div />;',
       'export const propTypes = {',
       '    a: PropTypes.any,',
+      '    b: PropTypes.string,',
       '    z: PropTypes.string,',
       '};',
       'export const defaultProps = {',
+      '    ...defaults,',
```

Comment:
instead of modifying this test, could you add a new one?

Reply:
Done